skills/KQAPro-KGQA/readme.md

```diff
@@ -0,0 +1,36 @@
+## Package
```

Comment:
Is this file required to check-in to the main repo? Seems just like your development comments?:)